### ISSUE TITLE
Visual Studio Code Insiders is not recognised as en editor

### ISSUE THREAD

**Issue by MSathieu** [desktop/desktop#3011]
I recently updated to the insiders version of Visual Studio Code, but now Github Desktop says that there are no editors installed.

### Version
**GitHub Desktop version:** [1.0.3]

**OS version:** [Windows 10]

### Steps to Reproduce

1. [Install Desktop and VSCode insiders]
1. [Try to open a project in the editor]

**Expected behavior:** [Open the project in VSCode]

**Actual behavior:** [Gives an error about no editor]
**Reproduces how often:** [100%]

**Comment by shiftkey** [comment]
@MSathieu not surprising - [these are the registry keys](https://github.com/desktop/desktop/blob/master/app/src/lib/editors/win32.ts#L23-L29) we use to find the VSCode install (64-bit or 32-bit), and [this is how we identify it's the right app](https://github.com/desktop/desktop/blob/master/app/src/lib/editors/win32.ts#L76-L80).

If they differ only by these details, it should be easy to extend it. Are you interested in digging up those details and contributing a PR? I'm happy to guide you along the way.

**Comment by MSathieu** [comment]
OK, I'll try @shiftkey . Should i try to detect it under ExternalEditor.VisualStudioCode or should i add something like ExternalEditor.VisualStudioCodeInsiders?

**Comment by lee-dohm** [comment]
@MSathieu Could you supply steps to reproduce the errant behavior here now that #3012 is merged?

**Comment by MSathieu** [comment]
Install Insiders and try to open a repo from Desktop. I don't know if the bug would be fixed when I reinstall desktop

**Comment by shiftkey** [comment]
@MSathieu if you look at the dev tools with verbose enabled, you'll see this message:

```
Command line interface for Visual Studio Code not found at 'C:\Program Files\Microsoft VS Code Insiders\bin\code.cmd'
```

This file is named `code-insiders.cmd` (probably to ensure it doesn't clash with an existing VSCode install), so the code needs to detect this as a fallback.

**Comment by MSathieu** [comment]
@shiftkey Should I make a new case, something like ExternalEditor.VisualStudioCodeInsiders?

**Comment by shiftkey** [comment]
@MSathieu maybe? I don't think we can have multiple enum values mapped to a single string (so we can separate the logic for `ExternalEditor.VisualStudioCodeInsiders` but make it appear in the UI as `'Visual Studio Code'`) so maybe we'll need to call it `'Visual Studio Code (Insiders)'`.

**Comment by shiftkey** [comment]
This was resolved by #3441
